eras/conway/impl/src/Cardano/Ledger/Conway/Governance/Internal.hs

```diff
@@ -266,11 +266,20 @@ instance EraPParams era => NoThunks (EnactState era)
 
 -- ========================================
 
+-- | `RatifyState` stores information about what will happen to the active
+-- governance actions at the next epoch boundary.
 data RatifyState era = RatifyState
   { rsEnactState :: !(EnactState era)
   , rsEnacted :: !(Seq (GovActionState era))
+  -- ^ Governance actions that are going to be enacted at the next epoch
+  -- boundary.
   , rsExpired :: !(Set (GovActionId (EraCrypto era)))
+  -- ^ Governance actions that are going to be removed at the next epoch
+  -- boundary, either due to expiring or because they would become invalid
+  -- after another governance action gets enacted or expired before it
   , rsDelayed :: !Bool
+  -- ^ True if all the governance actions will get delayed by one epoch at the
```

Comment:
This is such a small detail, but i found myself confused about it, so I will write a comment and feel free to ignore it! 
I believe the following is true: if one of the actions is a delaying one, then all of them will get delayed and this flag is set to True - is this correct?  
If it, then I think it would be clearer to say "True when all the governance actions will get get delayed", instead of "if", because with the "if"  I read it as:  all of them have to be delaying actions in order for  the flag to be set to true. 
I'm sure it's subjective - I'm more wanting to make sure I understand how it's actually working!

Reply:
I ended up rewriting the whole comment, thanks for the feedback!

Reply:
Awesome, thanks!